README.md

```diff
@@ -50,27 +50,58 @@ Update configurations for individual projects in **`installer/ansible/group_var/
 
 Please refer [SODA documentation](https://docs.sodafoundation.io/soda-gettingstarted/installation-using-ansible/) for the detailed installation steps and configuration options available.
 
+**Example#1:** To install Strato(Multi-cloud) and Dashboard using ansible installer.
```

Comment:
Please remove as Example:
**Installation steps for strato:** To install Strato(Multi-cloud) and Dashboard using ansible installer, please follow below steps.

Reply:
done